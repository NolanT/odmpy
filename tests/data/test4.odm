<?xml version="1.0"?>
<OverDriveMedia id="0fef5121-bb1f-42a5-b62a-d9fded939d50-425" ODMVersion="3.0.0.0" OMCVersion="3.0.0.0">
	<License>
		<AcquisitionUrl>https://ping.github.io/odmpy/test_data/test.license</AcquisitionUrl>
	</License><![CDATA[<Metadata>
	<ContentType>MP3 Audio Book</ContentType>
	<Title>Ceremonies For Christmas</Title>
	<SortTitle>Ceremonies For Christmas</SortTitle>
	<Series></Series>
	<Publisher>Librivox</Publisher>
	<ThumbnailUrl>https://ping.github.io/odmpy/test_data/cover.jpg</ThumbnailUrl>
	<CoverUrl>https://ping.github.io/odmpy/test_data/cover.jpg</CoverUrl>
	<Creators>
		<Creator role="Author" file-as="herrick, rober">Robert Herrick</Creator>
		<Creator role="Narrator" file-as="LibriVox, Volunteers">LibriVox Volunteers</Creator>
	</Creators>
	<Subjects>
		<Subject id="26">Fiction</Subject>
	</Subjects>
	<Languages>
		<Language code="en">English</Language>
	</Languages>
<Description>&Agrave;,&Aacute;,&Acirc;,&Atilde;,&Auml;,&Aring;,&AElig;,&Ccedil;,&Egrave;,&Eacute;,&Ecirc;,&Euml;,&Igrave;,&Iacute;,&Icirc;,&Iuml;,&ETH;,&Ntilde;,&Ograve;,&Oacute;,&Ocirc;,&Otilde;,&Ouml;,&Oslash;,&Ugrave;,&Uacute;,&Ucirc;,&Uuml;,&Yacute;,&THORN;,&szlig;,&agrave;,&aacute;,&acirc;,&atilde;,&auml;,&aring;,&aelig;,&ccedil;,&egrave;,&eacute;,&ecirc;,&euml;,&igrave;,&iacute;,&icirc;,&iuml;,&eth;,&ntilde;,&ograve;,&oacute;,&ocirc;,&otilde;,&ouml;,&oslash;,&ugrave;,&uacute;,&ucirc;,&uuml;,&yacute;,&thorn;,&yuml; &iexcl;,&cent;,&pound;,&curren;,&yen;,&brvbar;,&sect;,&uml;,&copy;,&ordf;,&laquo;,&not;,&shy;,&reg;,&macr;,&deg;,&plusmn;,&sup2;,&sup3;,&acute;,&micro;,&para;,&cedil;,&sup1;,&ordm;,&raquo;,&frac14;,&frac12;,&frac34;,&iquest;,&times;,&divide; fianc&eacute;e & friends &ldquo; &rdquo; &rsquo; [&nbsp;]</Description>
</Metadata>]]><DrmInfo>
	<PlayOnPC>1</PlayOnPC>
	<PlayOnPCCount>-1</PlayOnPCCount>
	<BurnToCD>1</BurnToCD>
	<BurnToCDCount>-1</BurnToCDCount>
	<PlayOnPM>1</PlayOnPM>
	<TransferToSDMI>1</TransferToSDMI>
	<TransferToNonSDMI>1</TransferToNonSDMI>
	<TransferCount>-1</TransferCount>
	<CollaborativePlay>0</CollaborativePlay>
	<PublicPerformance>0</PublicPerformance>
	<TranscodeToAAC>1</TranscodeToAAC>
<ExpirationDate>2099-01-01T00:00:00Z</ExpirationDate><Hash></Hash><Hash2></Hash2></DrmInfo><Formats><Format name="Medium Quality"><Quality level="Medium" /><Protocols><Protocol method="download" baseurl="https://ping.github.io/odmpy/test_data" /></Protocols><Parts count="1"><Part number="1" filesize="493921" name="Part 1" filename="book3/01_ceremonies_herrick_cjph_64kb.mp3" duration="01:01" /></Parts></Format></Formats><Source id="X"><Name>Library X</Name><WebsiteUrl>http://overdrive.com</WebsiteUrl><BannerUrl>http://overdrive.com/odmbanner.gif</BannerUrl><AccentColor>#eeeeee</AccentColor></Source><TransactionID>000-0000000-00000</TransactionID><EarlyReturnURL>https://ping.github.io/odmpy/test_data</EarlyReturnURL><DownloadSuccessURL>https://ping.github.io/odmpy/test_data</DownloadSuccessURL></OverDriveMedia>