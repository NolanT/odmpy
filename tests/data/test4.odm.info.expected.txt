Title:     Ceremonies For Christmas
Creators:  Robert Herrick (Author), LibriVox Volunteers (Narrator)
Publisher: Librivox
Subjects:  Fiction
Languages: English
Description:
À,Á,Â,Ã,Ä,Å,Æ,Ç,È,É,Ê,Ë,Ì,Í,Î,Ï,Ð,Ñ,Ò,Ó,Ô,Õ,Ö,Ø,Ù,Ú,Û,Ü,Ý,Þ,ß,à,á,â,ã,ä,å,æ,ç,è,é,ê,ë,ì,í,î,ï,ð,ñ,ò,ó,ô,õ,ö,ø,ù,ú,û,ü,ý,þ,ÿ ¡,¢,£,¤,¥,¦,§,¨,©,ª,«,¬,­,®,¯,°,±,²,³,´,µ,¶,¸,¹,º,»,¼,½,¾,¿,×,÷ fiancée & friends “ ” ’ [ ]

Format:    Medium Quality
* Part 1 - 01:01 (483kB)
